app/react/Layout/DocumentsList.js

```diff
@@ -242,6 +243,7 @@ DocumentsList.propTypes = {
     pathname: PropTypes.string,
     query: PropTypes.object,
   }),
+  CollectionViewer: PropTypes.instanceOf(Object),
```

Comment:
We changed this to func. Any of the other alternatives won't work because connected components are HOFs and for some reason the checker don't consider them as renderable.